v20.1/cockroachcloud-create-your-cluster.md

```diff
@@ -107,17 +106,17 @@ Let's consider a storage buffer of 50% to account for overhead and data growth.
 
 With the default replication factor of 3, the total amount of data stored is (3 * 450GB) = 1350 GB.
 
-To determine the number of nodes and the hardware configuration to store 1350 GB of data, refer to the table in [Step 2](#step-2-select-the-cloud-provider). We can see that the best option to store 1350 GB of data is 6 Large nodes.
+To determine the number of nodes and the hardware configuration to store 1350 GB of data, refer to the table in [Step 2](#step-2-select-the-cloud-provider). We can see that the best option to store 1350 GB of data is 9 Large nodes.
 
-Let's verify if 6 Large nodes meet our performance requirements of 2000 TPS. 6 Large nodes have (6*4) = 24 vCPUs. Since each vCPU can handle around 1000 TPS, 6 Large nodes can meet our performance requirements.
+Let's verify if 6 Large nodes meet our performance requirements of 2000 TPS. 6 Large nodes have (6*9) = 36 vCPUs. Since each vCPU can handle around 1000 TPS, 9 Large nodes can meet our performance requirements.
```

Comment:
> Let's verify if 6 Large nodes meet our performance requirements of 2000 TPS. 6 Large nodes have (6*9) = 36 vCPUs. Since each vCPU can handle around 1000 TPS, 9 Large nodes can meet our performance requirements.

Should this be identical to the text block in the 19.2 doc? It mentions 6 Large nodes, but I think it means to evaluate 9?

Reply:
Yes..fixed.